lib/elastic-search/requests.js

```diff
@@ -107,10 +107,10 @@ const currentDocument = async (bibId, index) => {
   const record = await client.search({
     index,
     body: {
-      query: { term: { uris: bibId } }
+      query: { term: { uri: bibId } }
```

Comment:
was this the bug?

Reply:
We no longer index `uris` (containing both bnum and all item nums), bug we've always indexed `uri` (just the bnum), so anywhere we're querying `uris` needs to change to `uri`